js/fileUtils.ts

```diff
@@ -1,31 +1,23 @@
 import invariant from "invariant";
-
-export interface MediaTags {
-  tags: {
-    artist: string;
-    title: string;
-    album: string;
-    picture: {
-      data: number[];
-      type: string;
-    };
-  };
-}
+import { IAudioMetadata, IOptions } from 'music-metadata-browser';
 
 type JsMediaTagsFile = string | ArrayBuffer | Blob;
```

Comment:
Can we rename this?

Reply:
What would you like to rename?

Reply:
I assume 'JsMediaTagsFile', I will change it to 'MediaDataType'